src/glrm.jl

```diff
@@ -268,7 +274,7 @@ function fit!(glrm::GLRM; params::Params=Params(), ch::ConvergenceHistory=Conver
     t = time() - t
     update!(ch, t, ch.objective[end])
 
-    return glrm.X, glrm.Y, ch
+    return glrm.X, glrm.Y, ch, alpha[1]
```

Comment:
Yeah, that's a good point. I'll put it into `ConvergenceHistory`. The reason it's helpful is when models take many iterations to converge it's useful to do checkpoints and restarts, but the restarts take longer to spin up (or break) without knowing at what stepsize the previous fitting was at.